src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+
+import "./index.css"; // 스타일 적용
```

Comment:
css는 동일한 우선순위라면 나중에 적용된 속성이 이전의 속성을 덮기 때문에, 컴포넌트 파일 이전에 import하는 게 안전합니다.
```suggestion
import "./index.css"; // 스타일 적용
import App from './App';
```